linker/Tests/Tests/ParseResponseFileLinesTests.cs

```diff
@@ -0,0 +1,63 @@
+using NUnit.Framework;
+using System.Collections.Generic;
+
+namespace Mono.Linker.Tests {
+	[TestFixture]
+	public class ParseResponseFileLinesTests {
+		[Test]
+		public void TestOneArg ()
+		{
+			Assert.That (TestParseResponseFileLines (@"abc", new string [] { @"abc" }), Is.True);
+		}
+
+		[Test]
+		public void TestTwoArgsOnOneLine ()
+		{
+			Assert.That (TestParseResponseFileLines (@"abc def", new string [] { @"abc", @"def" }), Is.True);
+		}
+
+		[Test]
+		public void TestTwoArgsOnTwoLine ()
+		{
+			Assert.That (TestParseResponseFileLines (@"abc
+	def", new string [] { @"abc", @"def" }), Is.True);
+		}
+
+		[Test]
+		public void TestOneSlashWithoutQuote ()
+		{
+			Assert.That (TestParseResponseFileLines (@"\", new string [] { @"\" }), Is.True);
+		}
+
+		[Test]
+		public void TestTwoSlashesWithoutQuote ()
+		{
+			Assert.That (TestParseResponseFileLines (@"\\", new string [] { @"\\" }), Is.True);
+		}
+
+		[Test]
+		public void TestOneSlashWithQuote ()
+		{
+			Assert.That (TestParseResponseFileLines (@"""x \"" y""", new string [] { @"x "" y" }), Is.True);
+		}
+
+		[Test]
+		public void TestTwoSlashesWithQuote ()
+		{
+			Assert.That (TestParseResponseFileLines (@"""Trailing Slash\\""", new string [] { @"Trailing Slash\" }), Is.True);
+		}
+
+		private bool TestParseResponseFileLines (string v1, string [] v2)
```

Comment:
This can be simplified to

```
		private void TestParseResponseFileLines (string v1, string [] v2)
		{
			var result = new Queue<string> ();
			Driver.ParseResponseFileLines (v1.Split ('\n'), result);
			Assert.That(result, Is.EquivalentTo(v2));
		}
```

And then each [Test] method is reduced down to for example,
`TestParseResponseFileLines (@"abc", new string [] { @"abc" });`

Less code and it will give better error message if a test were to fail.

Reply:
Fixed